README.md

```diff
@@ -140,16 +160,6 @@ That, of course, will send users to:
 
 (Or whatever Kubernetes release that docs branch is associated with.)
 
-## Branch structure
```

Comment:
I thought it repeated information given above in the document. Also, it mentions docsv2, and it's my understanding that we're going to shut down docsv2.